spacy/ml/models/coref.py

```diff
@@ -37,14 +37,11 @@ def build_wl_coref_model(
     except ValueError:
         # happens with transformer listener
         dim = 768
-    
+
     with Model.define_operators({">>": chain}):
         # TODO chain tok2vec with these models
-        # TODO fix device - should be automatic
-        device = "cuda:0"
```

Comment:
General comment about the device handling - is this unecessary because Thinc's `prefer_gpu` takes care of it? I notice there's no extra set up anywhere so I guess it just works?

Reply:
Yes, I kept it around in `wl-coref`, because of some hack, but `thinc` deals with it!